### DISCUSSION TITLE
How do I use multiple webviews in a single window?

### DISCUSSION THREAD

**Discussion by waryu-YND** [tauri-apps/wry#458] [Q&A] [answered]
I'd like to use multiple webviews in a single window.
Specifically, I'd like to build a window with a custom titlebar and another webview.
However, I don't know that, and could you teach me how to build it.

**Comment by amrbashir** [comment, marked as answer]
Multiple webviews are not possible yet. 

For a custom titlebar check this https://tauri.studio/en/docs/usage/guides/visual/window-customization#creating-a-custom-titlebar

**Reply by waryu-YND** [reply]
Thank you very much for answering my questions.

**Reply by ninetynin** [reply]
> Multiple webviews are not possible yet.
> 
> For a custom titlebar check this https://tauri.studio/en/docs/usage/guides/visual/window-customization#creating-a-custom-titlebar

Hey can u pls reshare the link.. as tauri moved from studio to app that url is broken

**Reply by amrbashir** [reply]
https://tauri.app/v1/guides/features/window-customization#creating-a-custom-titlebar

**Comment by abbiepup** [comment]
I'm wondering, is this in the current project scope, I mean planned for the future?

**Comment by niutech** [comment]
Multiple webviews are now implemented in tauri by [#8280](https://github.com/tauri-apps/tauri/pull/8280), see this example: https://github.com/tauri-apps/tauri/tree/dev/examples/multiwebview.
